tests/unit/modules/test_pyroute2.py

```diff
@@ -0,0 +1,12 @@
+
+from pyroute2 import IPDB
+from tests.support.unit import TestCase
+from salt.beacons.network_settings import ATTRS
+
+
+class Pyroute2TestCase(TestCase):
+
+    def test_interface_dict_fields(self):
+        with IPDB() as ipdb:
+            for attr in ATTRS:
+                self.assertIn(attr, ipdb.interfaces[1])
```

Comment:
I would not, definitively. All the interface attrs are several hundreds, and the list differs for different interface types. And the number increases with time. I don't think that the `salt` framework developers will be happy to maintain the full list.